src/containers/Login/components/AddProfileDetails.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UserSettings from 'containers/Settings/UserSettings';
+import { Modal, Text, Button } from 'components';
+import { PageCard } from 'explorer-components';
+import { actions } from '../reducer';
+
+const AddProfileDetails = props => {
+  const { visible, onClose, showForm, form } = props;
+
+  if (form) {
+    return (
+      <Modal visible={visible} size="large" dismissable onClose={onClose}>
+        <Modal.Header closable icon={['fal', 'faAddressCard']}>
+          <Modal.Heading>Profile Details</Modal.Heading>
+        </Modal.Header>
+        <Modal.Body>
+          <div className="container-fluid">
+            <div className="row center-xs">
+              <div className="col-xs-5">
+                Add some profile details about yourself, so that others on the
+                platform have an idea of your background and skillset, and can
+                find you elsewhere online.
+                <br />
+                <br />
+                We recommend you fill out as many details as possible, since
+                you'll need to submit a small transaction each time you change
```

Comment:
apostrophe's need to use the html 5 character: https://www.google.co.kr/search?q=html5+characters&rlz=1C5CHFA_enUS784US784&oq=html5+characters&aqs=chrome..69i57j0l5.2313j0j4&sourceid=chrome&ie=UTF-8

Reply:
👍